packages/ember-runtime/lib/system/core_object.js

```diff
@@ -455,7 +455,11 @@ CoreObject.PrototypeMixin = Mixin.create({
   },
 
   /**
-    Override to implement teardown.
+    Override to implement teardown. Will be called asynchronously.
+    This is scheduled for execution by the `destroy` method.
+
+    If you want to cancel a scheduled item with `Ember.run.cancel`, you should
```

Comment:
`scheduled` may lead to users believing `once` and `schedule` tasks are cancellable, but only `setTimeout` `later` `debounce` `throttle` are cancellable.